apps/omg/test/support/test_helper.ex

```diff
@@ -61,8 +61,8 @@ defmodule OMG.TestHelper do
   def generate_entity do
     {:ok, priv} = DevCrypto.generate_private_key()
     {:ok, pub} = DevCrypto.generate_public_key(priv)
-    {:ok, addr} = Crypto.generate_address(pub)
-    %{priv: priv, addr: addr}
+    <<_::binary-size(12), address::binary-size(20)>> = Crypto.hash(pub)
```

Comment:
hm, a leftover change? seems like `Crypto.generate_address(pub)` would still do the job here

Reply:
@pdobacz Ah good catch! Also removed some other duplication found :pray: